src/documents/tutorial-developerIntroduction/DeveloperIntroductionToInfusionFramework-OpeningRemarks.md

```diff
@@ -0,0 +1,598 @@
+page of the [Infusion Documentation wiki](http://wiki.fluidproject.org/display/docs/Infusion+Documentation)._ -->
+
+# Introduction
+
+Here we will boil down some of the basic concepts of Infusion for developers who may be interested in it, but uncertain of where to start. There's a lot going on in the framework, but grasping some core ideas helps a great deal in moving forward and learning.
+
+This introduction summarizes some topics (and leaves out others) that are gone into at length in the longer [Framework Concepts](/infusion/development/FrameworkConcepts.md) document; developers interested in greater detail about the guiding philosophy behind Infusion should consult that document as well.
+
+The introduction assumes you are familiar with the basics of [JavaScript](https://developer.mozilla.org/en-US/docs/Learn/JavaScript) development, and with using a browser's [developer tools](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools). You should also have some experience of the [jQuery](https://jquery.com/) JavaScript library that Infusion is built on top of.
+
+We will present an increasingly complicated "Hello, World!" implementation to demonstrate the core concepts, with live examples throughout using the [CodePen](http://codepen.io/) service.
+
+## Infusion's Core Concepts
+
+### Components
+
+Programs using Infusion structure their data, functionality and interaction as various components defined using a common syntax.
+
+Components are [JavaScript objects that have certain characteristics](/infusion/development/UnderstandingInfusionComponents.md). Components can be freely combined together to form new components using the [grades system](/infusion/development/ComponentGrades.md), or organized into relationships with one another via [subcomponents](/infusion/development/SubcomponentDeclaration.md).
+
+ There are three base components to be aware of that any new components you create with Infusion will likely have at their root:
+
+ * [`fluid.component`](/infusion/development/ComponentConfigurationOptions.md#options-supported-by-all-components-grades), the simplest type with support for events and public function definition (called [invokers](/infusion/development/Invokers.md) in Infusion).
+ * [`fluid.modelComponent`](/infusion/development/ComponentConfigurationOptions.md#model-components), which adds support for mutable state, model events, and state coordination and relay between different components.
+ * [`fluid.viewComponent`](/infusion/development/ComponentConfigurationOptions.md#view-components), which supplement model components with functionality to bind to the DOM of a web page.
+
+Component-based development emphasizes declarative configuration, loose coupling and flexible hierarchies. The framework manages component lifecycle and scoping and provides many supports to allow components to work in collaboration with one another.
```

Comment:
@waharnum, in your presentation at the community meeting, you went into a little more detail about the "loosely coupled" and "flexible hierarchies", comparing infusion to frameworks that use a more rigid derivation hierarchy.  You might want to expand this written version in a similar way.

Having said that, however, a reason that infusion is flexible is because it's based on JavaScript, and JavaScript itself promotes flexible loose coupling.  That is, it's easy in JavaScript to use composition to create objects with the desired functionality, and, while it's possible to create class-like hierarchies in JavaScript, it's not as straight forward.

Infusion capitalizes on JavaScript's inherent abilities in this regard, whereas some other frameworks/toolkits emulate class-based approaches and are more rigid as a consequence.